### PR TITLE
Debugger compatibility fixes, part 2

### PR DESCRIPTION
**Pull request by AnnaShaleva**

### CODE REVIEW

File: pkg/compiler/debug.go [hunk 1]
```diff
@@ -161,7 +164,7 @@ func (c *codegen) methodInfoFromScope(name string, scope *funcScope) *MethodDebu
 	return &MethodDebugInfo{
 		ID: name,
 		Name: DebugMethodName{
-			Name:      name,
+			Name:      strings.ToLower(string(name[0])) + name[1:],
```
Comment (fyrchik) [inline pkg/compiler/debug.go:167]: Probably not a popular case, but go allows to have multi-byte letters in function names. Not even sure we should support this, but this is not a hot path anyway.
https://goplay.tools/snippet/zHKtvoheyG1

Comment (AnnaShaleva) [reply pkg/compiler/debug.go]: Nice example!

File: pkg/interop/runtime/runtime.go [hunk 2]
```diff
@@ -29,7 +29,7 @@ func Log(message string) {}
 // part of contract's API to external systems, these events can be monitored
 // from outside and act upon accordingly. This function uses
 // `System.Runtime.Notify` syscall.
-func Notify(name string, arg ...interface{}) {}
+func Notify(name string, args []interface{}) {}
```
Comment (roman-khimov) [inline pkg/interop/runtime/runtime.go:32]: But why?

Comment (AnnaShaleva) [reply pkg/interop/runtime/runtime.go]: Just removed this commit, our notifications work fine on *up-to-date* master.

**Pull request merged**